### DISCUSSION TITLE
How to restore original update sources for manifests?

### DISCUSSION THREAD

**Discussion by MikeSesketcharba** [okibcn/ScoopMaster#109] [Q&A]
I used
```
gci ~/scoop/apps/*/current/install.json | % { 
    (gc $_) -Replace '(?<=bucket":\s+")[^"]+',".SM" |Set-Content $_ }
scoop update
scoop update *
```
to change the update sources for all my apps. But when I tried to install new apps, I kept getting hash fail errors. I removed .SM and added extras again, and those apps that failed the hash errors now are getting installed without problems.

How can I restore the original update sources for the apps I had installed before? I want to restore scoop to the state it was before .SM because it's not working out for me.

**Comment by hour-keeper** [comment]
Run it in powershell:
```powershell
$scoop_packages=$(scoop list).Name
scoop uninstall scoop
#Add official source here and remove .SM
scoop update && scoop install $scoop_packages
```
